### ISSUE TITLE
Keyboard Navigation

### ISSUE THREAD

**Issue by Isaac-hughes** [vicmans/ng-toggle-button#43]
For accessibility purposes the toggle should be tabbable and triggerable with keyboard events.

Great package by the way!

**Comment by vicmans** [comment]
You can do it, you can use Tab key to move through the controls and use the space bar to toggle, it is checkbox behavior.

**Comment by Isaac-hughes** [comment]
Thats great, is there a way to change the focus css so that it is consistant with my exisisting css?

**Comment by vicmans** [comment]
You can overwrite the class `.ng-toggle-focused`, it uses `box-shadow`, by default is:

```
.ng-toggle-focused {
  box-shadow: 0px 0px 4px 3px #999;
}
```
You can change the color, if it does not work use `!important`, this class is added when the element is focused with tab keyboard event, you can change it in your CSS as you need.

In a future version I think the focus color could be a component property
